modules/fluentd/s3.tf

```diff
@@ -64,6 +64,21 @@ data "aws_iam_policy_document" "logs_s3" {
       "${aws_s3_bucket.logs[0].arn}/*",
     ]
   }
+
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:PutLogEvents",
+      "logs:CreateLogGroup",
+      "logs:PutRetentionPolicy",
+      "logs:CreateLogStream",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams"
+    ]
+
+    resources = ["*"]
+  }
```

Comment:
suggestion: could we include the ssl policy that tzeyang implemented into this s3 resource?

Reply:
sure, i can add it